### PR TITLE
minor: Resolve Django 1.9+ allow_tags deprecation warning

### PR DESCRIPTION
**Pull request by ciarancourtney**
see https://docs.djangoproject.com/en/dev/releases/1.9/#id1

Resolves issue #78

### CODE REVIEW

File: safedelete/managers.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class SafeDeleteManager(models.Manager):
 
     :attribute _safedelete_visibility: define what happens when you query masked objects.
         It can be one of ``DELETED_INVISIBLE`` and ``DELETED_VISIBLE_BY_PK``.
-        Defaults to ``SOFT_DELETE``.
+        Defaults to ``DELETED_INVISIBLE``.
```
Comment (AndreasBackx) [inline safedelete/managers.py]: Thanks for noting, I guess we or I missed that.

**Comment by Gagaro** [comment]
Could you add a line in `CHANGES`?

Thanks.

**Comment by ciarancourtney** [comment]
FYI @Gagaro rebased CHANGES with master

**Comment by Gagaro** [comment]
Thanks @ciarancourtney, did you see my comment about `format_html`?

**Comment by ciarancourtney** [comment]
no?

**Comment by Gagaro** [comment]
What about now?

**Comment by ciarancourtney** [comment]
I don't see any comment about format_html

**Comment by Gagaro** [comment]
Strange. I'll put it here:

This should be:
```python
format_html('<span class="deleted">{0}</span>', obj_str)
```

**Comment by ciarancourtney** [comment]
ah ok np, will I rebase and keep it clean?

**Comment by Gagaro** [comment]
You don't have to, I'll squash the PR before merging it.

**Comment by Gagaro** [comment]
Thanks a lot :+1:

**Pull request merged**